src/config/sns.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "SNS",
+  "displayName": "Simple Notification Service",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "sns/%{DATA:region_id}/%{DATA:account_id}/%{USERNAME:resource_id}%{GREEDYDATA:failure_suffix}"
```

Comment:
**/**%{USERNAME:resource_id}%{GREEDYDATA:failure_suffix}
doesn't it need / between those two as well? 
Not 100% sure how it works, asking.

"logGroup": "sns/us-east-1/444000444/sample-sns-logs-generator/Failure",
"logGroup": "sns/us-east-1/444000444/sample-sns-logs-generator",

Reply:
I think you can't have this slash put here as it is optional. I think username will match all until slash or end. It if matched until slash then "failure_suffix" will contain "/Failure"